guide/attributes.md

```diff
@@ -1,16 +1,24 @@
 DOM Attributes
 --------------
+```
+login-input = Predefined value
+    .placeholder = example@email.com
+    .aria-label = Login input value
+    .title = Type your login email
 
-    login-input = Predefined value
-        .placeholder = example@email.com
-        .aria-label = Login input value
-        .title = Type your login email
-
-Use attributes when localizing more complex UI components such as DOM elements.
+```
 
 UI elements often contain multiple translatable messages per one widget. For
 example, an HTML form input may have a value, but also a `placeholder`
 attribute, `aria-label` attribute and maybe a `title` attribute.
 
 Another example would be a Web Component confirm window with an `ok` button,
 `cancel` button and a message.
+
+In order to prevent having to define multiple separate messages for representing
+different strings within a single element, FTL allows you to add attributes to
+messages.
+
+This feature is particularly useful in translating more complex widgets since
```

Comment:
you can click edit on the file and it'll allow you to fix it and create a PR from your fix :)